File: solutions/1013_fibonacci-number/solution.cpp
class Solution {
public:
    void fibo(vector<int>&v,int n){
        if (n==1){
            return ;
        }
        v.push_back((v[v.size() - 1]) + (v[v.size() - 2]));
        fibo(v, n - 1);
    }
    int fib(int n) {
        vector <int> v1 = {0,1};
        if (n==0){
            return 0;
        }
        if (n==1){
            return 1;
        }
        else{
            fibo(v1,n);
            return v1[v1.size()-1];
        }
    }
};

File: solutions/1013_fibonacci-number/solution.py
class Solution(object):
    def fib(self, n):
        a = 0
        b = 1
        if n == 1:
            return 1
        elif n==0:
            return 0
        for i in range(2,n+1):
            temp = a+b
            a = b
            b= temp
        return b
        